juno/assetlist.json

```diff
@@ -324,21 +324,21 @@
       {
         "description": "The native token cw20 for BlackHole on Juno Chain",
         "type_asset": "cw20",
-        "address": "juno1rdw3gumdz7zyjn2pev9ugxs765xlv6vtv6g3jt2lqw580zrchvjs66daca",
+        "address": "juno1w0ydaqem2xn0w8p28hvvqu9rmahhk2alg0qa9mt0nuuy4dk9yunst0yw0z",
         "denom_units": [
           {
-            "denom": "cw20:juno1rdw3gumdz7zyjn2pev9ugxs765xlv6vtv6g3jt2lqw580zrchvjs66daca",
+            "denom": "cw20:juno1w0ydaqem2xn0w8p28hvvqu9rmahhk2alg0qa9mt0nuuy4dk9yunst0yw0z",
             "exponent": 0
           },
           {
-            "denom": "blkh",
+            "denom": "bkhl",
             "exponent": 4
           }
         ],
-        "base": "cw20:juno1rdw3gumdz7zyjn2pev9ugxs765xlv6vtv6g3jt2lqw580zrchvjs66daca",
-        "name": "Blackhole",
-        "display": "blkh",
-        "symbol": "BLKH",
+        "base": "cw20:juno1w0ydaqem2xn0w8p28hvvqu9rmahhk2alg0qa9mt0nuuy4dk9yunst0yw0z",
+        "name": "BlackHole",
+        "display": "bkhl",
+        "symbol": "BKHL",
         "logo_URIs": {
           "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/juno/images/blkh.png",
```

Comment:
It could be very confusing to leave the image names the way they were. So, unless these blkh images uploaded to the chain reg are widely used among dapps directly, I'd recommend changing their name to bkhl.png and bkhl.svg. That's just a comment, and it can stay as is if you prefer.

Reply:
Or at the very least, clone the images while keeping the original as well